api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/audit/EventAuditorTest.java

```diff
@@ -187,4 +187,37 @@ void shouldLogEventForBalanceTooHighForPayment() {
                         entry(ClaimEventMetadataKey.PAYMENT_AMOUNT.getKey(), 0));
 
     }
+
+    @Test
+    void shouldLogClaimExpiredEvent() {
+        //Given
+        Claim claim = aValidClaim();
+
+        //When
+        eventAuditor.auditExpiredClaim(claim);
+
+        //Then
+        ArgumentCaptor<Event> eventArgumentCaptor = ArgumentCaptor.forClass(Event.class);
+        verify(eventLogger).logEvent(eventArgumentCaptor.capture());
+        Event actualEvent = eventArgumentCaptor.getValue();
+        assertThat(actualEvent.getEventType()).isEqualTo(EXPIRED_CLAIM);
+        assertThat(actualEvent.getTimestamp()).isNotNull();
```

Comment:
Can we assert change this to assert that the timestamp is greater than the beginning of the test.

Reply:
Done.